src/WebformCivicrmPostProcess.php

```diff
@@ -522,10 +522,27 @@ private function validateParticipants() {
     $this->loadEvents();
     // Add event info to line items
     $format = wf_crm_aval($this->data['reg_options'], 'title_display', 'title');
+    //Get the Event Fee Finacial Type Id if active, otherwise get the first active Financial Type Id
```

Comment:
Can you please fix the typo in the comment + add a space between //Get

Reply:
@KarinG - Comment is now fixed